src/node/desktop/src/core/file-path.ts

```diff
@@ -35,10 +35,10 @@ const homePathAlias = '~/';
 const homePathLeafAlias = '~';
 
 function normalizeSeparators(path: string, separator = '/') {
```

Comment:
Add a documentation comment to this method describing what it does, /** */.

Reply:
Alright